CMakeLists.txt

```diff
@@ -101,6 +101,17 @@ if (ENABLE_GOMA)
   set_compiler_launcher(ENABLE_GOMA gomacc)
 endif ()
 
+if(ENABLE_EXTRA_SPEED)
+	if(MSVC)
+		#enable parallel builds
```

Comment:
Please fix this comment: it doesn't explain what /Qpar is actually doing. Microsoft documentation states the following:

Enables the Auto-Parallelizer feature of the compiler to automatically parallelize loops in your code.

Reply:
sorry my fault, this line should be removed. Visual Studio's comment was irritating in this sense, stating: "Enable Parallel Code Generation"